static/assets/js/entity-reference/app.js

```diff
@@ -0,0 +1,38 @@
+(function(){
+    "use strict";
+
+    var app = angular.module('entityReferenceApp', []);
+
+    app.controller('EntityReferenceCtrl', ['$http', EntityReferenceCtrl]);
+
+    app.component('entityReference', {
+        templateUrl: '/assets/views/entities.html',
+        controller: 'EntityReferenceCtrl as ctrl',
+        bindings: {
+            title: '@',
+            getStartedUrl: '@',
+            swaggerUrl: '@'
+        }
+    });
+
+    function EntityReferenceCtrl($http) {
+        this.title = 'Entity Reference';
+        this.showErrorMessage = false;
+
+        var handleSuccess = function(response) {
```

Comment:
if you do these as named functions, then you can put them below the call to $http since they're automatically forward declared in the scope where they're created (iow, you won't ever get an undefined if you do it like `function handleSuccess(...)`) and this will make the code a little easier to read.